File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow.cpp

#include "SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow.h"
#include "SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  std::string SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow.cs
/// </filename>
namespace commands
{
    public class SelectRowCommand_ViewModelMyStringRowHandleListViewElementsRow
    {
        private string HeaderLabelText;

        public virtual string getHeaderLabelText()
        {
            return this.HeaderLabelText;
        }

        private string RowHandle;

        public virtual string getRowHandle()
        {
            return this.RowHandle;
        }
    }
}
